web/spacewalk-web.changes

```diff
@@ -1,3 +1,4 @@
+- Fix inconsistent HMR and key usage bug (bsc#1180580)
```

Comment:
This changelog is often read by normal user of Uyuni/SUSE Manager. Try to write a message which they can understand. 

Like "fixing display error of Package State page" or something similar.